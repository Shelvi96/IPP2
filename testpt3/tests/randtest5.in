NEW ID

77>5:2:45
:;351>4541::
9826592>42;101;8:
;974>684;96977
216;80>:49291:
0957146>092:::58
:636;34>1:02986:66
2092104;:>;2
6925>:86108
9>030
546;5>358
4>;2;
;48;:90:>19349416:4
8;>435:6;14;
4:85:>4;8;75
0354>1739:0
9095;7>17206:
29>0537:0
47:1::1>22
511>83
3529651;>921;:89;
049>6187234
2;>801224336
684483840>;6:60
2275886920>570119999
51980>6;
01>;;2753
95646797>74105:
:>799726
182:03;>:77952
46884:5>0071;97
38;5>39914
:1;95:52>7;8;37024
;;8;58669>:694
618372;>2:1
371375;:2;>803
;029;85>3792987
33;62890:>8:0
9;3386>11
0:91:4;1>;;1;
1:89890>8419
4:3;368>454
47966:;86>975681;544
77:147:1;8>:9
47>1602
98>;8
34::;04>760:2:83
534:180660>50204
285>340
7>6
2305026024>3
73;>03:8
83513>7
7958>9:095
8053>43
01701392>9
89;55>2
:;47;41709>;13;242
70198:05>:76973
6711;>3306
6;8>683795724
::7>57
591404;4>532150
0>;565
47;3>5;420
:4>:8833:
82;812>872476873;
5;03761369>96
;5417916>91:59580:
55637298>624670;2
4>6
28;31;3;>0453371;87
550946;075>915024757
21;50>9102697
2176800969>02961982
5>5685
62>18187217;
91>3:547577
6492>:04614823:
31202134>730633
716>;:5:4166
81>130
43032449;>16083060::
88;89;9>;11
644:059>770
30>3289:48:67
2:13470>9066016109
640222>214:970
:>:267
06>6006
64>:6705:
9064097979>648406
049>930721046
81612>:1367;
4;0>48:65
7170>0630;:
5346>:6:4:14
5>84:
;39;8190>12769;
09;259>1;15
5318:27>38
9158067568>8
5024>;4276:58
44535;01>7041
0219675>6:118;;8
345;98;>762;0
3>7086675
0:4>90315006
394:342:74>04
6392376>5:1:7
26:5337:>98:1
:>:40;:7
8>77656870
571>60:
;34116:55>1456;4192
87752960>8;43
68:8>747
11796847>;3:508:14
:7>4820:
8>52:4;507
967>4;725;
3:37>:;807
994211>0137123
955848;>7996
9>;777108
12;03133>89;4
2310>;6259155
5126356::>2193558711
65582327>81127
;67:79>90021836
9;5:684180>172
@068734884;946708;56552652161358
@;86603256390;182:67781436;49848;135160:0704:55043734187773978561836635:3971202:392;2:6:92;424235
@93162439561;481:830659
@45:1;16531:97614;;937646276018995;:558282499::211349987345491
@:;49499:9;2794935;6:08
@7;24736029935:438493510
@63683632:97515:32;:79225:30381324
@30062:398866;859764102406051727:2884636
@0003995849:8;609;636828:286
@303318;254:1509872:6814874::44;7923:2240661;73;65945:0
@:838077741:64424:991801292:652;3:2;::69670
@34777490415:38891:940;7211400454043;14456949946273:77528668;2536::6;3281;0:19434023775763855104322
@943:37884500964851339;0654:870;84670178640:1:69
@85;9803142:3614;7;;8:;63058:;:6;368;60126
@:002;7127322:6774653:09637:498779;106676:988377;102343:8
@00090;:;1::7549606171506;8236:5:397395228495874055;:2442079:928008315:3560;7;77
@0
@333;97:69809005853:8841;;06;9;:46573061161:6682;;08;59;419328562:2920:36060627917544
@390
@42121807977825089:24734508339306:28;20:;79;223:311818460094:54
@3
@46444401767;4:49:5:23:7569:31155;:136581;3085958272::93861;;69897:01302
@77;0480:838643238666333:5730793246707;33
@;1;736310:471609918;83772821;5826791153
@3;18:19;66:3162364499037;504:7:5731983831;:29093414567
@90462087;;1074829;0;848;0135;311767::3912211:1471
@:91:114873;88;6318629052678:7969:7839408;
@8867;73::5277863
@1453743:17216::249600429:1977051924877602120;47711759137206:44;
@:71520:8100487;10;:90597975638;275;9599:1929952941;9604471231158:89366
@777945;6905:;927879;02881598;72:317;6758726:;1
@882;1589:1;6468292:185:4413344044785047:968145
@2;5772547:;2336;;17953870645:1140:;;1570362:182121:75:394;720874663;
@;;751972;08523:0:;49669655549478465944374;067265627498;628:;06;84460118550407:9114521807
@;6999515366560:64:56;
@;4819464152:6;3;475;4;32:7431365;2:8:44;96139932801004623:585315505842;68116:80:95:11434604;4681:69
@894826606627;0808386419076594330480667:45005409046:0;74:5979421822281056053
@958136522107;;971:484649;1836;84889;7759858;897:03641260424254:24
@5
@7;078975:67:1572;1373;:0386780;33;;;0:4;83998438563095506;0233:621:33623:;4:
@7656263;385280:;2457682125;69128872256582;3231559244;651;8;091091
@3619778:133:80044;:9519817:54308915805:18138
@4467284
@5180319629376403253493617409
@54:60295;3;0:;:6;1944117;6237;8476:78;5064083;3244;990887;3623::909:;2:9;292549:88;5075;:
@5543323406727:4542950:1376:1512;66210:54446423975:0906080215778153261029491742615::5825886
@4569;949:4;080174819147:014122;:857;7;857;940;;8;5601521;:65193
@320:6071;953894;6:3775064;;986:392572185261:7511340290
@1;020;030697027645;60471;;510:261781695672584460:56:9148499470
@9;7:5873390
@54552:317497865569378:95:50260548;1;15411182
@5796:46163;33597;5427734
@4945;7:699953:;;122;4::1
@00174402095;;088124;5340934;555
@::52698:62:56:
@74::2162:262737
@5:284;
@7529287474256708:66011;2794767314::7:9361:3753771177779286121836655538;86284:3;;976868:22
@537016::;57215819:91260066840:7566:;40:3
@59:2:00815:75;53;93:8186;7;356828:56:92;25:;83274594653952
@:08187:6979116302;2177;068731984189:774565:7;57549;48286:
@244:05693280;2519968639575774::662467:734733104:1;:8675105843:21
@67756291556:94;06;615;5:;259461;5183398122;4:
@44:3572:9606754989:4011828:1084406;956720087904516958212:38:3477:6430394390098523673109;752:995
@71204;488809
@:34827928992;73047;59;2182
@82350:9918:3575;:4629126350852
@258;971:8193:801;58161748554:7:44682194125912921226042547:
@5572:5:47;8012128;995;3679;04112608856445556;:179:871311141956336;4;58425194;2314;8531:89;52599385
@66;8;0983;;43;4169;26370042457;;
@:7525158024;65588;2373435084;7:1333048882001999:9;1469731480365:18:9975;7509317058
@;:77;37475956643738626;;14659711500447030:0:4453;592;05054563:;8;;4765::7:0;756:;30:8:352
@3533522111:335148092;8:3;61637664999343451;8710352:8280565;95:7979;312;:3;624;:958674:1:7
@;026:2395
@;426719;47197;4;99142098:7:28
@:;0557:6;250;0;738944381;70:185;;985563409431735
@01;7:56998:66153:208:49;00073967;:21
@1:010:7183;2304093018059600471;83493699200970179
@0;9486207;70:7869;1687:989125;;9281:63:52:655632540642700825;565573026994761198;289:34
@345
@826185907394;5:4600521244:2;7
@33294:65218;:;4043080697:555848;7:0406920:621225572653541:19
@196492444;829:3;419;775:01326197:4;;687:;3481076
@498;6:372613315;046808771791376;82334:::44185:;935:7692;53
@0:6370:2890718015;536:518
@77;;;31
@;09926:754359142825171739;6867650772551:9547:096327:86559;547397;81423330;2:;046639212;;29
@1040;5187832065:9964315:37161:16;562:029050139;16:91;23615073116;091033017544552723:875:4957
@:62:732:67;104999;8437024261;54:360:279020264386342:;243925871:3831::2:4
@4278:53375629:3;7;:8
@18536759;85585458;1157:7:161033
@308272;:2442:08:;14802428:385;99;:;;421::5
@091;8635:9105;755676:5599;:7980:5516;839941739003761432561549566;7023340;573207:;6;49614760806
@317582102709104;3:3;09184784;6:6;90741;:0;723317686:5;61765
@085;99:529331294:2;4;:249068637:34111;6302:147:2:1:9;0514;1:780:41354:9507:878;5:9612;2
@349:4989306
@23063322
@57175394
@7;58926:39294459:84423:195606215272390
@0939576696:286493133;64403634654706504;9;:374;5708236:8:26265;34;19;5914;44789689848:060:8
@7;60;;12823510908668360;54:97:02:6619836:;;4384363;994160;4794:;380048:7;9;2
@75328500665920:72:2;:67:52965;418;358796663072;10102881122;86792;4;7412:75:6;:83306;0;
@318;7302245:864:69101304612:15647276:7004624463:8831;
@5980;256:406;:1;39952245045758325;25183443:723;51827:64329:751:210760229604083510;8::010
@086568
@:893:;080480855051:3543:8;41940049370334;04;51;:625;699605722;26051
@05
@39236329:421233840862;0945299;6080374:4626;5:65671491465112208291:490036663819
@8::536487529534646216280124131833:97957
@2:6;126900:636:80
@94;
@;30029
@957737095995770560602243173294;62;69:6738403051:57
@;12321:82381977655101:09:92079:6:594:011796485:1245425547742868:3:2
@:7;58594:;903289055741908523799548:
@1;53;7869489254675;7312::04706866599097:27;818201578;1
@91;429403:942;005;4144745132051160:09205421711;653;9762575;;20194;155622
@49590328203060033;6707330916;3:34754:704;8550518873127428
