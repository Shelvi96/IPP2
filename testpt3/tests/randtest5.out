18146895179811134589
0
13145865594
0
15459769006
409582584048
14447679030184378368
5053053006841380864
551353010000000
0
6365648957288218624
0
16244987370376573496
8216614110486331392
0
0
0
0
0
0
0
0
696078942616508094
5053053006841380864
0
12033618204333965312
8216614110486331392
2753
0
0
11642930936659574784
11642930936659574784
0
0
150109938
0
0
0
0
0
0
15728664738510148840
0
6431111100000000
0
0
0
0
0
0
12762990237735452672
553417321100
553417321100
5642244010000000000
25
211
0
0
5296403860967915520
0
0
0
5581930263149019136
0
0
195815653102344
14676084511751798784
0
0
15148406240826964319
0
0
0
0
55233341210000000
16375088245119123456
7144632429582483456
16375088245119123456
0
0
1223007042645983232
0
16375088245119123456
0
20347613363473
0
0
0
0
65121701
12546134052891056184
6582929054055721224
0
0
0
0
0
0
14256145805852278784
0
0
0
0
0
0
0
0
0
10202720594173775061
35911
0
0
15276209936040722432
0
1062929378944539224
0
13855660688692754541
0
